### ISSUE TITLE
Check subject after 26th of june

### ISSUE THREAD

**Issue by Ala-Na** [Reaster0/ft_transcendence#124]
Because many students shared their 42 API keys on their public github repo, 42's staff said they will modify ft_transcendence and others concerned projects subjects on the 26th.
I guess those modifications should only concerned the API stuff and maybe a line about leaving our .env inside our git repo will be eleminatory. But further check is necessary.

**Comment by Reaster0** [comment]
yeah from what i understand we will just have to add the env file in the correction and not pushed with all the files in the vogsphere right?

**Comment by Reaster0** [comment]
nothing right now beside this part stress me a little about not putting .env files in it
![Capture d’écran 2022-06-27 à 13 09 57](https://user-images.githubusercontent.com/73885131/175928294-233119f1-c4c6-44cb-bb6f-ffe34a5bf0a2.png)

and also, to make it work accross multiple macs, we will need to make a api key for a special computer at school and stick with it for the entier correction, since we're bound to this private ip

**Comment by Ala-Na** [comment]
I re-read discord/slack message and modification should happne on 29th, not 26th. Here what they say about it: 'Starting 29th, if an API key is found inside a git repo, outside of a .env file created locally during evaluation, evaluation should stop.'. I guess we should avoid pushing our .env and create it locally, as asked. If we start corrections before the 29th, we can always show the discord/slack message as proof.
